test/AutoExpandbleMap/AutoExpandble_map_utest.cpp

```diff
@@ -2,46 +2,49 @@
 
 #include "Map_test.h"
 
-bool is_succeed(const DiscretePoint2D& Cell_coor, MapParams expected_param) {
+bool test_expansion_by_cell_update(const DiscretePoint2D& Cell_coor, MapParams expected_param) {
```

Comment:
Guys, i think we can increase line length limit up to 100. I mean {80-100] is OK